src/components/dropdown-filter/dropdown-filter.js

```diff
@@ -328,23 +345,29 @@ class DropdownFilter extends Dropdown {
         html = [original];
 
     if (this.state.open && this.props.create) {
-      let text = 'Create ';
+      const textNew = I18n.t('dropdown_filter.new_text', { defaultValue: ' New' });
+      let createText = I18n.t('dropdown_filter.create_text', { defaultValue: 'Create' });
 
-      if (this.state.filter) {
-        text += `"${this.state.filter}"`;
+      if (this.props.createText) {
+        createText = this.props.createText;
+      } else if (this.state.filter) {
+        createText += ` "${this.state.filter}"`;
       } else {
-        text += 'New';
+        createText += textNew;
       }
 
       html.push(
-        <a
+        <Link
+          icon={ this.props.createIconType || 'add' }
+          iconAlign='left'
           className='carbon-dropdown__action'
           data-element='create'
+          data-component='create'
```

Comment:
this should be `data-element` - as it is a part of the `DropdownFilter` component.

`data-component` - defines a component
`data-element` - defines a part of a component
`data-role` - defines a specific component in an app